File: app/src/main/java/com/example/getfood/ui/foodmenu/FoodMenuDisplayActivityMvpPresenter.java
package com.example.getfood.ui.foodmenu;

import com.example.getfood.ui.base.BaseMvpPresenter;

public interface FoodMenuDisplayActivityMvpPresenter<V extends FoodMenuDisplayActivityMvpView> extends BaseMvpPresenter<V> {

    String getUserEmail();

    void sendPasswordResetEmail();

    void signOutUser();

    void openCart();
}

File: app/src/main/java/com/example/getfood/ui/foodmenu/FoodMenuDisplayActivityMvpPresenter.kt
package com.example.getfood.ui.foodmenu

import com.example.getfood.ui.base.BaseMvpPresenter

interface FoodMenuDisplayActivityMvpPresenter<V : FoodMenuDisplayActivityMvpView?> : BaseMvpPresenter<V> {
    val userEmail: String?
    fun sendPasswordResetEmail()
    fun signOutUser()
    fun openCart()
}